docs/snippets/ov_dynamic_shapes.py

```diff
@@ -56,20 +58,20 @@
 
 #! [set_input_tensor]
 # For first inference call, prepare an input tensor with 1x128 shape and run inference request
-Input_data1 = np.ones(shape=[1,128])
-infer_request.infer([input_data1])
+input_data1 = np.ones(shape=[1,128])
+infer_request.infer({"Parameter_319": input_data1})
 
 # Get resulting outputs
-Output_tensor1 = infer_request.get_output_tensor()
-Output_data1 = output_tensor.data[:]
+output_tensor1 = infer_request.get_output_tensor()
+output_data1 = output_tensor1.data[:]
 
 # For second inference call, prepare a 1x200 input tensor and run inference request
-Input_data2 = np.ones(shape=[1,200])
-infer_request.infer([input_data2])
+input_data2 = np.ones(shape=[1,200])
+infer_request.infer({"Parameter_319": input_data2})
```

Comment:
should we provide better names for operations in test model?

Reply:
updated and hide under the variable